### PR TITLE
Feature add/remove

### PR DESCRIPTION
**Pull request by AKeeganDev**
This Pull request features 4 main components:

Add task capabilities - Which adds a task that is typed into the input field at the top of the list. To use please hit ENTER or click on the submit arrow

Modify() - which saves any changes you make to a list element in real-time (no other actions required)

Delete - When you click on a task to edit it you will see a delete icon (trashcan) on the element you are editing. Click that button to remove it from the task list

Reload - All tasks appear back on the screen from your last session. It saves all changes typed into the task as well as the order that they were in when you left the screen.

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @AKeeganDev, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] In README File, In `Getting started` section should contain the necessary code to run the project locally, For ex:-
```
git clone https://github.com/AKeeganDev/To_Do_List.git
cd To_Do_List
```

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/pagemanager.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      this.checkBoxes = document.querySelectorAll('.check-box');
+    }
+    this.loadCheckBoxListeners();
+    this.loadTaskItemInputListeners();
+    this.loadUpdateButtonListeners();
+  }
+
+  addTask = () => {
+    const counter = this.taskList.length + 1;
+    const newTask = new Task(this.taskInput.value);
```
Comment (MrBrN197) [inline src/modules/pagemanager.js]: - [ ] [OPTIONAL] you can add a check that the `this.taskInput` is empty and simply return. realistically, you wouldn't want to add an item that has no text.
```suggestion
  addTask = () => {
    if(!this.taskInput.value) return;
    const counter = this.taskList.length + 1;
    const newTask = new Task(this.taskInput.value);
```

**Review by MrBrN197 — APPROVED** [review]
## Hi, @AKeeganDev  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/pagemanager.js (hunk 1 above)
Comment (AKeeganDev) [reply src/modules/pagemanager.js]: @MrBrN197 Good idea. I'll do that Thank you for the suggestion and the review. Cheers!

**Pull request merged**